cev_eris.dme

```diff
@@ -2143,6 +2144,7 @@
 #include "code\modules\recycling\disposal.dm"
 #include "code\modules\recycling\sortingmachinery.dm"
 #include "code\modules\research\circuitprinter.dm"
+#include "code\modules\research\debug.dm"

```

Comment:
debug still here though

Reply:
oh no that's deliberate.

This is a debug object that maxes out research levels when fed into the destructive analyzer, there's no way to obtain it ingame. It has potential usefulness for admin shenanigans, and it will be useful to have active as a debugging tool in future

Reply:
If you say so.